pkg/command/context.go

```diff
@@ -64,15 +63,15 @@ const (
 	deactivatingPlugin                  = "Deactivating plugin '%v:%v' for context '%v'"
 
 	invalidTargetErrorForContextCommands = "invalid target specified. Please specify a correct value for the `--target` flag from 'kubernetes[k8s]/mission-control[tmc]'"
-	invalidContextType                   = "invalid context type specified. Please specify a correct value for the `--type/-t` flag from 'kubernetes[k8s]/mission-control[tmc]/application-engine[tae]'"
+	invalidContextType                   = "invalid context type specified. Please specify a correct value for the `--type/-t` flag from 'kubernetes[k8s]/mission-control[tmc]/tanzu'"
 )
 
 // constants that define context creation types
 const (
 	contextMissionControl     ContextCreationType = "Mission Control"
 	contextK8SClusterEndpoint ContextCreationType = "Kubernetes (Cluster Endpoint)"
 	contextLocalKubeconfig    ContextCreationType = "Kubernetes (Local Kubeconfig)"
-	contextApplicationEngine  ContextCreationType = "Application Engine"
+	contextTanzu              ContextCreationType = "Tanzu"
```

Comment:
Would like suggestions/comments to rename `Tanzu` to `Tanzu control plane`.
( This name `Tanzu` appears in the prompt is user tries to create context using prompts, so feeling little odd showing just `Tanzu` as context)

Reply:
Based on the comment [above](https://github.com/vmware-tanzu/tanzu-cli/pull/562/files#r1369314786). I will keep `Tanzu` as is.